manta-pay/abstract-protocol.tex

```diff
@@ -458,19 +663,19 @@ \subsection{Abstract Cryptographic Schemes}
             \State $B \gets A + \remainder(B, N)$
         \EndWhile
         \State $S \gets \PrepareZeroes_\sk(N, B, Z, P)$ \Comment{Use $Z$ and \Mint{s} to make $B$ go up to $N$ in size.}
-        \State $R \gets \BuildReceiver_\sk(\rk, S)$
+        \State $R \gets \BuildReceiver_\sk(\addr, S)$
         \State $[c, zs...] \gets \BuildAccumulatorAndZeroes_\sk(S)$
         \State \textsf{return} $P + \TransferPost(\Transfer([], S, [R, c, zs...], []))$
     \EndProcedure
 \end{algorithmic}
 \end{algorithm*}
```

Comment:
1. Sampling method is chosen by the wallet implementer to be efficient. @bhgomes give a description of the one we are currently using
2. For the new protocol, one `ak` is enough to decrypt all assets (because of diversifier).